sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt

```diff
@@ -23,20 +23,19 @@ internal fun Navigation(
   val navigator = rememberCircuitNavigator(backstack)
 
   Box(Modifier.fillMaxSize()) {
-    for (record in backstack.take(2).asReversed()) {
-      key(record.key) {
-        when (val screen = record.screen) {
-          is GalleryScreenKey -> {
-            GalleryScreen(
-              key = screen,
-              navigator = navigator
-            )
-          }
-          is MediaViewerScreenKey -> {
-            MediaViewerScreen(
-              key = screen
-            )
-          }
+    val record = backstack.first()
```

Comment:
There's currently a bug where you can still keyboard the gallery screen when the media viewer is brought up, I'm not familiar enough with circuit to know what the right fix was so just changed it to render only the current page while testing. That does break the swipe away animation since it won't render the behind page until the animation completes

Reply:
> There's currently a bug where you can still keyboard focus the gallery screen when the media viewer is brought up

hmmmmmm I wonder if I can fix this by preventing touch/focus events from leaking behind the active screen.

Reply:
On my side project where I'm using a custom navigation solution I solved this by not rendering the behind page unless you were in the middle of the transition.